src/ks_log.c

```diff
@@ -291,9 +282,9 @@ static void default_logger(const char *file, const char *func, int line, int lev
 				ks_json_add_string_to_object(json, "thread", tbuf);
 			}
 
-			ks_json_add_string_to_object(json, "file", file);
-			ks_json_add_string_to_object(json, "func", func);
-			ks_json_add_number_to_object(json, "line", line);
+			ks_json_add_string_to_object(json, "calling-file", file);
+			ks_json_add_string_to_object(json, "calling-func", func);
+			ks_json_add_number_to_object(json, "calling-line", line);
```

Comment:
This is a default logger- I'd leave this alone.  If the goal is trying to fit into some specific infrastructure, then either this lib needs to allow that via configuration, or you hook in a custom logging callback that does this.

Reply:
I guess we can make it configurable or fscb has to implement a custom logger just to change those names, because "file" isn't liked by elastic, it complains about the field existing.
Let me know what you'd like to do with this.